src/com/epam/javatrainings/webpage/WebPageDownloader.java

```diff
@@ -0,0 +1,71 @@
+package com.epam.javatrainings.webpage;
+
+import java.io.*;
+import java.net.URL;
+
+public class WebPageDownloader {
+
+    private WebPageDownloader() {
+
+    }
+
+    public static void downloadPageBy8kbBuffer(String url) {
```

Comment:
let's not expose internal implementation in method name, DownloadPage is enough I think

Reply:
Also you can add bufferSize as a method parameter